entries/a/adac.de.json

```
@@ -0,0 +1,15 @@
+{
+  "Raisin": {
```

Shouldn't this be "ADAC"?

```suggestion
  "ADAC": {
```